api/v1/v1_test.go

```diff
@@ -41,7 +41,7 @@ func runEnd2EndJob(pullRefs, seedRefs []string) ([]string, error) {
 		}
 	}
 
-	pushConfig := PushConfig{Registry: registryContainer.Hostname()}
+	pushConfig := PushConfig{Registry: registryContainer.Hostname(), PathSeparator: "/"}
```

Comment:
A default `PathSeparator` "/" should be passed for tests as long as we use stock docker registry image in our suite

Reply:
Default should already be / if you do not pass anything.

Reply:
For CLI yes as it handled by CLI args / flags package. For direct API usage in tests - no. ;)